speech/unit_tests/test__gax.py

```diff
@@ -50,39 +50,24 @@ def test_constructor(self, mocked_stub, mocked_cls, mocked_channel):
         from google.cloud.speech._gax import OPERATIONS_API_HOST
 
         credentials = _make_credentials()
-        mock_cnxn = mock.Mock(
-            credentials=credentials,
-            spec=['credentials'],
-        )
+        mock_cnxn = mock.Mock(credentials=credentials, spec=['credentials'])
         mock_client = mock.Mock(
-            _connection=mock_cnxn,
-            _credentials=credentials,
-            spec=['_connection', '_credentials'],
-        )
+            _connection=mock_cnxn, _credentials=credentials,
+            spec=['_connection', '_credentials'])
 
         speech_api = self._make_one(mock_client)
         self.assertIs(speech_api._client, mock_client)
-        self.assertIs(
-            speech_api._gapic_api,
-            mocked_cls.return_value,
-        )
+        self.assertIs(speech_api._gapic_api, mocked_cls.return_value)
 
         mocked_stub.assert_called_once_with(
-            mock_cnxn.credentials,
-            DEFAULT_USER_AGENT,
-            operations_grpc.OperationsStub,
-            OPERATIONS_API_HOST,
-        )
+            mock_cnxn.credentials, DEFAULT_USER_AGENT,
+            operations_grpc.OperationsStub, OPERATIONS_API_HOST)
```

Comment:
You really went wild with these style changes? One might argue in this case you've made the code **less readable**.

No need to change, just a remark.